packages/@expo/cli/package.json

```diff
@@ -155,6 +155,7 @@
     "@types/wrap-ansi": "^8.0.1",
     "@types/ws": "^8.5.4",
     "devtools-protocol": "^0.0.1113120",
+    "expo-atlas": "0.0.14",
```

Comment:
Hardcoded for now, need to answer these questions:
  - Do we want to keep optionally ship Atlas -- right now it's optional, only activated when installed
  - How do we want to ensure the version match -- for now, it's just a `typeof require('expo-atlas/cli').createExpoAtlasMiddleware === 'function'` check.

Reply:
We are shipping atlas as a separate library, but with integration inside the Expo CLI. See the `AtlasPrerequisite` and `attachAtlasAsync` functions.